tests/app/letters/test_letter_utils.py

```diff
@@ -13,6 +33,70 @@ def test_get_bucket_prefix_for_notification_valid_notification(sample_notificati
     ).upper()
 
 
+@freeze_time(FROZEN_DATE_TIME)
```

Comment:
I think that the freeze time masks the issue with using a transient date

Reply:
I would introduce a test specific for that, and maybe fix it in another PR